src/wolff_producer.erl

```diff
@@ -537,13 +544,15 @@ do_handle_kafka_ack(BaseOffset,
                calls := Calls,
                attempts := Attempts}}, NewSentReqs} = queue:out(SentReqs),
     AttemptedBefore = Attempts > 1,
-    inc_sent_success(Config, 1, AttemptedBefore),
+    inc_sent_success(Config, length(Calls), AttemptedBefore),
```

Comment:
this was the must fix.

Reply:
yes.

Reply:
sorry, one queue item is one call.
but one sent-request (a batch of Kafka messages) can contain messages from multiple calls.

Reply:
I see, thanks